public/build/js/client.03b9f659.js.LICENSE.txt

```diff
@@ -0,0 +1,62 @@
+/*
```

Comment:
Did you mean to add all these license files?

Reply:
no, these can be removed if not needed